### ISSUE TITLE
RefreshHandler doesn't set default SigningAlgorithm

### ISSUE THREAD

**Issue by buildscientist** [appleboy/gin-jwt#109]
Hello, 

There's a bug in the RefreshHandler using the example code in the README. 

If I do not set the SigningAlgorithm in my GinJWTMiddleWare struct - RefreshHandler passes a nil value to the JWT library. 

See lines [392-393](https://github.com/appleboy/gin-jwt/blob/b98b512ed67b501994175b49a1adfc709b8702d5/auth_jwt.go#L392-L393) of auth_jwt.go

The LoginHandler calls MiddlewareInit - lines [209-210](https://github.com/appleboy/gin-jwt/blob/b98b512ed67b501994175b49a1adfc709b8702d5/auth_jwt.go#L209-L210) to set the default SigningAlgorithm to HS256.

**Comment by appleboy** [comment]
@buildscientist Thanks for report this issue. maybe we should add the following check in `RefreshHandler`.

```go
	// Initial middleware default setting.
	if err := mw.MiddlewareInit(); err != nil {
		mw.unauthorized(c, http.StatusInternalServerError, mw.HTTPStatusMessageFunc(err, c))
		return
	}
```

**Comment by appleboy** [comment]
closed #134. You can try the following example:

```go
	r := gin.New()

	r.POST("/login", auth.LoginHandler)

	group := r.Group("/auth")
	group.Use(auth.MiddlewareFunc())
	{
		group.GET("/hello", helloHandler)
		group.GET("/refresh_token", auth.RefreshHandler)
	}
```
